electrum/commands.py

```diff
@@ -353,10 +353,12 @@ async def setconfig(self, key, value):
             cv.set(value)
 
     @command('')
-    async def make_seed(self, nbits=None, language=None, seed_type=None):
+    async def make_seed(self, nbits=None, language=None, seed_type=None, extra_entropy: str = None):
```

Comment:
ThomasV remarked:
> re extra_entropy, I think that option should not be named that way. it is misleading. some users might believe that their seed will have more bits of entropy than what is passed in nbits

Reply:
Alternative names:
- `addentropy`, `add_entropy`, `add_to_entropy`,
- `mixin_entropy`, `mix_in_entropy`,
- `user_entropy`

Out of these, I prefer `add_entropy` or `user_entropy`, and re behaviour, we can describe how the value is used here:
https://github.com/spesmilo/electrum/blob/6de55a43fe0ad531716378c945f42d8a57b97efa/electrum/commands.py#L1442